src/components/Type2020/Blueprint.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
+ *
+ *
+ * Name proposals
+ * --------------------------
+ * Type refers to the current soon to be "legacy" Type components.
+ * We need something to differentiate from our current Type component while transitioning.
+ *   - Type2020 (working prototype in this doc & files)
+ *   - Type2020 / TwentyType
+ *   - TypeNew / NewType
+ *   - TrendyType
+ *   - ???
+ *
+ * Should we append the new name to the end of Type2020 components like below?
+ * TitleSmall vs TitleSmall2020
+ *
+ * Should we rename Type to TypeLegacy or similar?
```

Comment:
This would be nice, sure, but seems impractical from the perspective of how much scope it will add.